src/test/java/seedu/address/logic/parser/FilterClientCommandParserTest.java

```diff
@@ -17,7 +17,7 @@ public class FilterClientCommandParserTest {
     @Test
     public void parse_nameSpecified_success() {
         // Test for valid phone number
-        String userInput = FilterClientCommand.COMMAND_WORD + " " + PREFIX_NAME + name;
+        String userInput = String.format(" %s%s", PREFIX_NAME, name);
```

Comment:
Refactoring string concatenation to String.format makes the code much cleaner and readable!